web/src/features/Dashboard/components/DashboardVersionCard.tsx

```diff
@@ -168,6 +166,15 @@ const DashboardVersionCard = (props: Props) => {
   );
   const history = useHistory();
   const params = useParams<KotsParams>();
+  const { selectedApp } = useSelectedApp();
+  const {
+    data: newAppVersionWithInterceptData,
+    error: latestDeployableVersionErrMsg,
+  } = useNextAppVersionWithIntercept(selectedApp?.slug || "");
+  const { latestDeployableVersion } = newAppVersionWithInterceptData || {};
+
+  const { data: isHelmManagedResponse } = useIsHelmManaged();
+  const { isHelmManaged = false } = isHelmManagedResponse || {};
```

Comment:
this is getting a little messy. I'm still into the idea of returning the `react-query` api as is just so there's a standard interface but may need to come up with a cleaner way to query some of our nested data responses. maybe we can un-nest them with `react-query`